src/bin/cql-stress-cassandra-stress/settings/command/mod.rs

```diff
@@ -71,6 +73,7 @@ impl Command {
             Command::CounterWrite => "Multiple concurrent updates of counters.",
             Command::CounterRead => "Multiple concurrent reads of counters. The cluster must first be populated by a counterwrite test.",
             Command::Mixed => "Interleaving of any basic commands, with configurable ratio and distribution - the cluster must first be populated by a write test.",
+            Command::User => "Interleaving of user provided queries, with configurable ratio and distribution",
```

Comment:
As of now, there is no need to require that.

In c-s, `user` mode has no validation logic for the inserted data - this means that populating the cluster is not necessary.

However, looking at c-s code, it seems that someone started implementing data validation. Unfortunately, it seems to fail during parsing, and obviously, it's not documented. There is an idea to add such feature to cql-stress, though. Do you think it's better to add such requirement right now, even though it will only be required in the future?

Reply:
In general, yes; let's introduce requirements early, so that developers never have to change their habits and/or code once they get used to them.